taskcluster/ci/build/kind.yml

```diff
@@ -104,6 +104,18 @@ jobs:
         treeherder:
             symbol: nightly(B)
 
+    android-test:
+        attributes:
+            nightly: false
+        run-on-tasks-for: [github-push]
```

Comment:
Could we just make the `fennec-nightly` job run on `github-push` instead of duplicating it?

Reply:
The issue with this is that it somewhat conflates the term "nightly" which should only be built once per day - not on each commit.

Reply:
Technically, that's exactly we want – we want a nightly build to test against for each commit.

Maybe we can call it "nightly simulation" to disambiguate?

Reply:
Nightly simulation sounds good! I'll make the change.

Reply:
I'm on the fence on this one too. If I'm not mistaken, we had the same kind of issues on the gecko repo. I don't know the details of it. @Callek, do you think we should implement something like a `shippable` flavor?